### DISCUSSION TITLE
Paperless says files are missing

### DISCUSSION THREAD

**Discussion by harmlessdonkey** [jonaswinkler/paperless-ng#1139] [Support] [answered]
I logged into Paperless today and it's giving all the following errors in the logs for all of the documents I have added
"[ERROR] [paperless.sanity_checker] Original of document 50 does not exist."

All the files are in the same place they have always been. I'm not sure what has changed or how to fix it. Can I associate the documents somehow?

**Comment by jonaswinkler** [comment, marked as answer]
Well its exactly what it says. Paperless can't find the original file of th document with the ID 50. Try opening the document in the web UI (you can paste the ID into the URL) and try to download the original file (click the little arrow on the download button). Chances are high that it won't work.

**Reply by harmlessdonkey** [reply]
Thanks, so all the documents have this error. All the underlying files are still there they haven't moved and I haven't changed the docker config file so it should still have access to the location where they are stored.

**Comment by krikkert** [comment]
If you run bash in the container (docker-compose exec webserver bash) and cd to the media folder (/usrc/src/paperless/media), does it throw any errors when listing the files (ls)?
